src/main/webapp/js/maps-page-loader.js

```diff
@@ -2,42 +2,58 @@ const MAP_ELEMENT_ID = 'map';
 const MAP_NIGHT_TYPE_ID = 'night'
 /**
  * This function is meant to create the map, set the centre and zoom, add a new map style and add markers.
- * Map styles are different modes that will be displayed to the user (The user can switch around with button)
- * Added night mode (the static configs for that style is in extraMapStyles.json)
+ * Map styles are different modes that will be displayed to the user
```

Comment:
these changes aren't from this PR, right? i'm confused.

Reply:
I think they are there because I probably did something wrong with the git stuff. Somewhere and somehow after I git pull origin, my intellj started to complain of errors and I did a bunch of stuff and I think these might have been added...I'll go check it out and fix this. Thanks for pointing it out